core/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+class ProductDetailTag(models.Model):
+    banner = models.TextField()    
+    productDetail = models.TextField()
+    sec3 = models.TextField()
+    sec5 = models.TextField()
+    sec6 = models.TextField()
+    purchaseView = models.TextField()
```

Comment:
column 은 카멜케이스 보다는 소문자로 작성 부탁 드립니다.
```suggestion
    purchase_view = models.TextField()
```